src/components/Form/story.tsx

```diff
@@ -0,0 +1,73 @@
+import React from 'react';
+import { storiesOf } from '@storybook/react';
+
+import Form from '.';
+import Readme from './README.md';
+
+storiesOf('Uncategorized/Form', module)
+  .addParameters({
+    readme: {
+      sidebar: Readme,
+    },
+  })
+  .add(
+    'Overview',
+    () => (
+      <Form
+        initialValues={{ nameInput: 'james r. cat' }}
+        // eslint-disable-next-line no-alert
+        onSubmit={(formValues) => alert(JSON.stringify(formValues))}
```

Comment:
This is where, if no submit handler is provided, I think falling back to default form behavior would make sense.